tests/unit/helpers/name-pattern-validator.test.js

```diff
@@ -0,0 +1,17 @@
+import namePatternValidator from '../../../source/helpers/name-pattern-validator';
+import {IMAGE_NAME_FORMAT_HASH, IMAGE_NAME_FORMAT_DATE, IMAGE_NAME_FORMAT_ORIGIN} from '../../../source/constants';
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [A space is required before '}'.](https://app.codacy.com/gh/pass-team/windows-10-lock-screen-image-extractor/pullRequest?prid=5038858)